skada/_mapping.py

```diff
@@ -100,6 +106,7 @@ def transform(
         X_adapt, _ = source_target_merge(
             X_source, X_target, sample_domain=sample_domain
         )
+        X_adapt = self.alpha * X_adapt + (1 - self.alpha) * X
```

Comment:
The alpha transformation should be done on X source not on X_adapt ? I think it is the same but right now for target you are doing: alpha * X_target + (1 -alpha) * X_target =X_target